subgraphs/aggregator/tests/helpers/utils.ts

```diff
@@ -0,0 +1,117 @@
+import { Address, BigInt, Bytes, ethereum, Wrapped } from "@graphprotocol/graph-ts";
+import { newMockEvent } from "matchstick-as";
+import { OrderFulfilled } from "../../generated/Seaport/Seaport";
+
+/**
+ * @param amount amount in ETH (i64)
+ * @param decimals number of decimal (optional)
+ * @notice Parse the amount into a BigInt instance of the amount of wei.
+ */
+export function parseEther(amount: i64, decimals: u8 = 18): BigInt {
```

Comment:
Maybe it should exist at the root of the repo?

Reply:
Yes, actually this is not even used, removed.